docs/source/methods/avas.rst

```diff
@@ -168,6 +168,241 @@ We then use this guess of active orbitals to compute the CASCI energy:::
+                                   # plane 2 with C atoms #9, #10, and #11, and N atoms #1 and #2.
+  - [['C1-4'], ['C1-2', 'C5-6']]   # plane 1 with the first four C atoms of the molecule,
+                                   # plane 2 with C atoms #1, #2, #5, and #6.
+                                   # Two planes share C1 and C2!
+
+This syntax follows the one used by :code:`SUBSPACE`: ::
+
+  - ["C"]              # all carbon atoms
+  - ["C","N"]          # all carbon and nitrogen atoms
+  - ["C1"]             # carbon atom #1
+  - ["C1-3"]           # carbon atoms #1, #2, #3
+  - ["C(2p)"]          # the 2p subset of all carbon atoms
+  - ["C(1s)","C(2s)"]  # the 1s/2s subsets of all carbon atoms
+  - ["C1-3(2s)"]       # the 2s subsets of carbon atoms #1, #2, #3
+  - ["Ce(4fzx2-zy2)"]  # the 4f zxx-zyy orbital of all Ce atoms
+
+.. note::
+    It is very important to include a complete set of p orbitals in :code:`SUBSPACE` if π planes are defined.
+    Otherwise, the code will follow the directions given by :code:`SUBSPACE`.
+
+.. tip::
+    The code is flexible enough to treat double active spaces (e.g., double π or double d-shell).
+    For example, the double-π active space of formaldehyde can be obtained via ::
+
+      set forte {
+        minao_basis        double-shell
+        subspace           ["C(2p)","C(3p)","O(2p)","O(3p)"]
+        subspace_pi_planes [["C","O","H"]]
+        avas               true
+        avas_diagonalize   true
+        avas_cutoff        0.5
+      }
+
+    Here I prepare a basis called "double-shell.gbs", which includes the 2p and 3p orbitals of C and O atoms.
+    You can also prepare your own MINAO basis by truncating the the cc-pVTZ or ANO-RCC-VTZP basis sets.
+
+For a more realistic example, consider the following iron porphyrin related molecule:
+
+.. image:: images/FeP.png
+    :width: 600
+    :align: center
+    :alt: An iron porphyrin complex.
+
+This molecule contains two π systems, namely, porphyrin and imidazole.
+Also, the porphyrin is not a perfect plane anymore.
+The following snippet selects all 3d orbitals of Fe, 3p orbitals of S,
+and all p orbitals that are perpendicular of porphyrin and imidazole rings. ::
+
+  set forte {
+    avas                true
+    avas_diagonalize    true
+    avas_cutoff         0.5
+    minao_basis         cc-pvtz-minao
+    subspace            ["Fe(3d)","C6-25(2p)","N(2p)","S(3p)","C1-3(2p)"]
+    subspace_pi_planes  [["Fe","C6-25","N3-6"], ["N1-2","C1-3"]]
+  }
+
+The AVAS output selects exactly 37 orbitals we wanted. ::
+
+  ==> AVAS MOs Information <==
+
+    ---------------------
+                        A
+    ---------------------
+    DOCC INACTIVE     106
+    DOCC ACTIVE        22
+    SOCC ACTIVE         0
+    UOCC ACTIVE        15
+    UOCC INACTIVE     462
+    ---------------------
+    RESTRICTED_DOCC   106
+    ACTIVE             37
+    RESTRICTED_UOCC   462
+    ---------------------
+
+  ==> Atomic Valence MOs (Active Marked by *) <==
```

Comment:
Could we condense this table with lines that say "..."? Would the point we want to make be sacrificed if we shrink it?